examples/test_cc_shared_library/starlark_tests.bzl

```diff
@@ -6,13 +6,19 @@ load("//examples:experimental_cc_shared_library.bzl", "for_testing_dont_use_chec
 def _linking_suffix_test_impl(ctx):
     env = analysistest.begin(ctx)
 
-    target_under_test = analysistest.target_under_test(env)
     actions = analysistest.target_actions(env)
 
-    for arg in reversed(actions[1].argv):
-        if (arg.find(".a") != -1 or arg.find("-l") != -1) and "/" in arg:
-            asserts.equals(env, "liba_suffix.a", arg[arg.rindex("/") + 1:])
-            break
+    found = False
+    for action in actions:
+        if action.mnemonic != "CppLink":
+            continue
+
+        found = True
+        for arg in reversed(action.argv):
```

Comment:
@oquenchil I'm not sure if the order change here is important or not, but there are now different actions and index 1 didn't seem like what we wanted anymore

Reply:
Ahh, missed this. I thought this PR was just fixing formatting.

I will rewrite all these tests today not to rely on the index. I will get back to you.